pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: flutter_localized_locales
 description: Flutter plugin which provides locale code to name mappings for 563 locales.
-version: 2.0.0
+version: 2.0.1-nullsafety.0
```

Comment:
```suggestion
version: 2.0.1
```

Reply:
This change is necessary I think in order for pub to pick up that the package has been moved to nullsafety, giving it the `null-safe` badge and forcing users to explicitly specify this version in their pubspec order to use it